### ISSUE TITLE
How to run OpenFortranParserXML-0.6.1.jar successfully

### ISSUE THREAD

**Issue by youcoding98** [mbdevpl/open-fortran-parser-xml#13]
I use the command: `java fortran.ofp.FrontEnd --class fortran.ofp.XMLPrinter \ --output output.xml --verbosity 0~100 input.f `torun,
but failed. Can it run successfully with only Java environment and no Python environment?Is this a separate command?
I ran this command successfully ：`java fortran.ofp.FrontEnd --verbose main.f90`

**Comment by mbdevpl** [comment]
Yes, it can be ran independently. Indeed, in the readme in https://github.com/mbdevpl/open-fortran-parser-xml#how-to-run I wrote `--verbosity 0~100` but also below that it is explained that it is a value between 0 and 100. 0~100 in this case means correct value range. If you provide a number between 0 and 100 inclusive, it will work as expected.

If you'd find some bits of the readme confusing and would like those bits to be clarified, I'd welcome a PR :)

**Comment by youcoding98** [comment]
Thanks！Now I can run it successful. I read this program's code,find that  `FortranParser2008_FortranParserBase.java` as the concrete file to achieve AST. For Example `main_program()` ,we can find it in this  `FortranParser2008_FortranParserBase.java`.But I can't find this java file in this github program. This only is find in  OpenFortranParse-0.8.5.jar. So I want to know how to achieve this file, Or How to override OpenFortranParse to modify this function？ Or How to use the Github program to create the jar?
Thank you very much!
